### PR TITLE
Adding Artifact Hub to replace Helm Hub in docs.

### PR DESCRIPTION
**Pull request by bridgetkromhout**
This PR updates website content that is not auto-generated.

After https://github.com/helm/helm/pull/8860 is merged and that version is released, we'll also need to update the generated docs on the website.

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

File: content/en/docs/intro/using_helm.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ https://hub.helm.sh/charts/presslabs/wordpress-...  v0.6.3        v0.6.3      Pr
 https://hub.helm.sh/charts/presslabs/wordpress-...  v0.7.1        v0.7.1      A Helm chart for deploying a WordPress site on ...
 ```
 
-The above searches for all `wordpress` charts on Helm Hub.
+The above searches for all `wordpress` charts on Artifact Hub.
```
Comment (hickeyma) [inline content/en/docs/intro/using_helm.md:58]: "on the Artifact Hub."?

File: content/en/docs/topics/v2_v3_migration.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ during migration:
      namespace anymore
 2. Chart repository updated:
    - `helm search` now supports both local repository searches and making search
-     queries against Helm Hub
+     queries against Artifact Hub
```
Comment (hickeyma) [inline content/en/docs/topics/v2_v3_migration.md:28]: "against the Artifact Hub"?

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for updating this @bridgetkromhout.

Small potential changes inline. Also, do you mind looking at other files that might need a change:

```console
$ find . -name "*" -type f | xargs grep -i "helm hub"
./i18n/en.toml:other = "Visit the [Helm Hub](https://hub.helm.sh) to explore charts from numerous public Helm repositories."
./themes/helm/layouts/index.html:          <p><a href="https://hub.helm.sh"><img src="/img/helm-hub.jpg" alt="Helm Hub" class="is-centered"></a></p>
./content/en/docs/intro/using_helm.md:- `helm search hub` searches [the Helm Hub](https://hub.helm.sh), which
./content/en/docs/intro/using_helm.md:The above searches for all `wordpress` charts on Helm Hub.
./content/en/docs/topics/v2_v3_migration.md:     queries against Helm Hub
./content/en/docs/community/history.md:Museum, and later the Helm Hub.
./content/en/docs/helm/helm_search.md:they can be stored including the Helm Hub and repositories you have added. Use
./content/en/docs/helm/helm_search.md:* [helm search hub](helm_search_hub.md)	 - search for charts in the Helm Hub or an instance of Monocular
./content/en/docs/helm/helm_search_hub.md:search for charts in the Helm Hub or an instance of Monocular
./content/en/docs/helm/helm_search_hub.md:Search the Helm Hub or an instance of Monocular for Helm charts.
./content/en/docs/helm/helm_search_hub.md:The Helm Hub provides a centralized search for publicly available distributed
./content/en/docs/helm/helm_search_hub.md:Helm Hub. You can find it at https://github.com/helm/monocular
```

File: content/en/docs/topics/v2_v3_migration.md (hunk 2 above)
Comment (bridgetkromhout) [reply content/en/docs/topics/v2_v3_migration.md:28]: The question of whether to use the definite article or not is one that features prominently in "CNCF" vs "The CNCF" as well. If we're going to say "Helm Hub" and "DockerHub" without a definite article, then we probably don't need "The Artifact Hub"?

**Comment by bridgetkromhout** [comment]
The stuff under `./content/en/docs/helm/` is generated via `helm docs --type markdown`. I submitted a PR to update it in the code (https://github.com/helm/helm/pull/8860/files) after which we'll be able to use the documented process (https://github.com/helm/helm-www#updating-the-helm-cli-reference-docs) to update it on the website.

File: content/en/docs/topics/v2_v3_migration.md (hunk 2 above)
Comment (hickeyma) [reply content/en/docs/topics/v2_v3_migration.md:28]: Good point. No need for definite article then.

File: content/en/docs/intro/using_helm.md (hunk 1 above)
Comment (hickeyma) [reply content/en/docs/intro/using_helm.md:58]: answered in https://github.com/helm/helm-www/pull/884#discussion_r501849243

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @bridgetkromhout

**Comment by bridgetkromhout** [comment]
> other files that might need a change

Updated the front page as @hickeyma suggested. If approved, I will suggest this change to the various translations.

**Comment by bridgetkromhout** [comment]
Generated docs updated in https://github.com/helm/helm-www/pull/921.

**Pull request merged**